src/Form/UserPickupRequestForm.php

```diff
@@ -344,9 +344,16 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
             $submit_message = ($cancel_holds ? 'Your requests were successfully canceled' : 'Pickup appointment scheduled for ' . date('F j', strtotime($pickup_date)) . ' at ' . $locations->{$branch});
             $messenger->addMessage($submit_message);
  
-            $user = \Drupal\user\Entity\User::load(\Drupal::currentUser()->id());
-            $uid = $user->id();
-            $url = \Drupal\Core\Url::fromRoute('entity.user.canonical', ['user'=>$user->id()]);
+            // redirect to the user's account page (if user is logged in) or to the front page
+            $currentUserId = currentUser()->id();
```

Comment:
Looks like you are looking at an older commit to the branch because the latest commits on the branch are since the variable naming changes were made to adhere to the Drupal coding conventions